File: 1895.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int alice = 0;
        int bob = 0;

        Scanner sc = new Scanner(System.in);
        String[] inputs = sc.nextLine().split(" ");
        int n = Integer.parseInt(inputs[0]);
        int t = Integer.parseInt(inputs[1]);
        int limite = Integer.parseInt(inputs[2]);

        for(int i = 0; i < n - 1; i++) {
            int s = sc.nextInt();
            if(Math.abs(t - s) <= limite) {
                if(i % 2 == 0) {
                    alice += Math.abs(t - s);
                } else {
                    bob += Math.abs(t - s);
                }
                t = s;
            }
        }

        System.out.println(alice + " " + bob);
    }
}

File: 1895.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };
let alice = bob = 0;
let [n, t, limite] = prompt().split(' ');
for (let i = 0; i < parseInt(n) - 1; i++) {
    let s = parseInt(prompt());
    if (Math.abs(parseInt(t) - s) <= parseInt(limite)) {
        if (i % 2 === 0) {
            alice += Math.abs(parseInt(t) - s);
        } else {
            bob += Math.abs(parseInt(t) - s);
        }
        t = s;
    }
}
console.log(alice, bob);
